util/patch/patch.go

```diff
@@ -291,6 +296,15 @@ func (h *Helper) calculateChanges(after client.Object) (map[string]bool, error)
 		return nil, errors.Wrapf(err, "failed to unmarshal patch data into a map")
 	}
 
+	// log whether a patch is created. If enabled log the full patch.
+	if len(patchDiff) == 0 {
+		log.Info(fmt.Sprintf("No changes for %s", tlog.KObj{Obj: h.after}))
+	} else {
+		if options.LoggingEnabled {
```

Comment:
The LoggingEnabled option should probably be slightly renamed. We are logging in any case the flag just decides if we also log the patch.

I would have expected something else based on the name of the option.

Reply:
Duplicate finding - concurrent review with Fabrizio. I'll resolve :)

Reply:
Agreed, are you happy with the idea of "PatchLoggingEnabled"?